config/locales/fr.yml

```
@@ -2,6 +2,11 @@
 fr:
   admin:
     rdvs:
+      message:
+        confirm:
+          simple_cancel: Annuler le rendez-vous ?
+          cancel_with_notification: Annuler le rendez-vous ? Les usagers seront notifiés de cette annulation.
```

Notifier, dans le sens “être prévenu” est un anglicisme :)

```suggestion
          cancel_with_notification: Annuler le rendez-vous ? Les usagers seront informés de cette annulation.
```